.github/workflows/auto_pr_test.yml

```diff
@@ -15,67 +15,33 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
 
+    # We use a pre-fab container with PETSc and MPICH included.
+    # NOTE: Docker containers may only work for Linux builds currently.
+    container:
+      image: jedbrown/petsc:latest
+      env:
+        # These PETSC variables depend where PETSc is installed
+        # See https://github.com/jedbrown/jedbrown-dockerfiles/blob/main/petsc/Dockerfile
+        PETSC_DIR: /usr/local/petsc
+        PETSC_ARCH: mpich-int32-real-opt
```

Comment:
These environment variables are exported in the container so I'd assume this job would inherit them. Also, it's a prefix install so `PETSC_ARCH` should be empty and `PETSC_DIR=/usr/loca/petsc/mpich-int32-real-opt`.

Reply:
Cool, I'll remove them. And thanks for the petsc-root image.